### DISCUSSION TITLE
Hacktoberfest

### DISCUSSION THREAD

**Discussion by brylie** [CiviWiki/OpenCiviWiki#1076] [General]
A thread to discuss Hacktoberfest.

**Comment by vishnudas-bluefox** [comment]
hey am loooking for some kind of beginner friendly issue:)

**Reply by brylie** [reply]
Welcome! Perhaps you can take a look at the following bugs to see whether they have been resolved?

- https://github.com/CiviWiki/OpenCiviWiki/issues/121
- https://github.com/CiviWiki/OpenCiviWiki/issues/1080

**Comment by gorkemarslan** [comment]
@brylie and @AbhijithGanesh, you completed 4 tasks in Hacktoberfast and were rewarded, right? Otherwise, let me know and help you. 😄

**Reply by AbhijithGanesh** [reply]
I didn't do it for the rewards , and I got my 4 PRs I reckon. Thank you though

**Reply by gorkemarslan** [reply]
We're all here to contribute to the project and the community. But who doesn't want to get a nice t-shirt? 😄

**Reply by AbhijithGanesh** [reply]
I'm also lucky enough to be in India cuz it's shipped it from here

**Reply by brylie** [reply]
I didn't register for Hacktoberfest but instead had a good time reviewing the pull requests :smiley:
